packages/ember/tests/routing/basic_test.js

```diff
@@ -321,8 +321,8 @@ QUnit.test('Model passed via renderTemplate model is set as controller\'s model'
   equal(jQuery('p:contains(emberjs)', '#qunit-fixture').length, 1, 'Passed model was set as controllers model');
 });
 
-test('Renders correct view with slash notation', function() {
-  setTemplate('home/page', compile('<p>{{view.name}}</p>'));
+QUnit.test('Renders correct view with slash notation', function() {
```

Comment:
This entire test should be removed or renamed.

This test explicitly tests the "correct view" is rendered. Since views are gone in Canary, we no longer need to test that they render. Obviously `App.HomePageView` is no longer used here, so that variable can do away. The title of this test is also wrong, as your update drops the use of a view.

We should keep a test that `Renders correct template with slash notation` and perhaps even `Renders correct controller with slash notation`. Perhaps those are elsewhere in this file, perhaps we should add them.


/cc @krisselden @rwjblue